packages/now-cli/src/commands/teams/switch.js

```diff
@@ -40,13 +40,23 @@ export default async function({ apiUrl, token, debug, args, config }) {
 
   const stopUserSpinner = wait('Fetching user information');
   const client = new Client({ apiUrl, token });
-  const user = await getUser(client);
+  let user;
+  try {
+    user = await getUser(client);
+  } catch (err) {
+    if (err.code === 'NOT_AUTHORIZED' || err.code === 'TEAM_DELETED') {
+      console.error(error(err.message));
```

Comment:
shouldn't `error` already write to stderr?

Reply:
You would think so but it does not https://github.com/zeit/now/blob/master/packages/now-cli/src/util/output/error.ts

Reply:
@AndyBitz I tried to make this PR touch the fewest amount of lines so as to avoid conflicts with your PR. That's why I didn't do createOutput() here or else I would have to pass `debug` through.